### PR TITLE
Update libs

### PR DESCRIPTION
**Pull request by jarl-dk**
This is an update (rebase of https://github.com/spockframework/spock/pull/568)

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,5 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.8-bin.zip
+distributionSha256Sum=a1242e4db8f979998796b1844e608c2acf8f8f54df518bbb3d5954e52253ba71
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.11-all.zip
```
Comment (szpak) [inline gradle/wrapper/gradle-wrapper.properties]: Why not 2.13?

Comment (szpak) [inline gradle/wrapper/gradle-wrapper.properties]: And why `-all` instead of `-bin`?

**Comment by oreissig** [comment]
I updated my original PR, raising library versions to the now current ones.

**Comment by szpak** [comment]
LGTM

**Pull request merged**